spec/models/user_search_spec.rb

```diff
@@ -51,10 +51,22 @@ def search_for(*args)
     _samantha = Fabricate(:user, username: 'samantha')
     group.add(sam)
 
-    results = search_for("sam", group: group)
+    results = search_for("sam", groups: [group])
     expect(results.count).to eq(1)
   end
 
+  it 'allows filtering by multiple groups' do
+    group_1 = Fabricate(:group)
+    sam = Fabricate(:user, username: 'sam')
+    group_2 = Fabricate(:group)
+    samantha = Fabricate(:user, username: 'samantha')
+    group_1.add(sam)
+    group_2.add(samantha)
```

Comment:
Hmm in `users_controller_spec` we're saving the group after adding a user but we're not doing so here. Might want to ensure the consistency here :grin: